superset-frontend/packages/superset-ui-core/test/number-format/factories/createMemoryFormatter.test.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { NumberFormatter, createMemoryFormatter } from '@superset-ui/core';
+
+describe('createMemoryFormatter()', () => {
+  it('creates an instance of MemoryFormatter', () => {
+    const formatter = createMemoryFormatter();
+    expect(formatter).toBeInstanceOf(NumberFormatter);
+  });
+  it('formats bytes in human readable format with default options', () => {
```

Comment:
Oh, one more thing, we try to steer away from nested tests. So instead of doing `describe('foo)` with many `it('bar')` and `it('baz')`, just do individual `test('foo bar')` and `test('foo baz')`

Reply:
sure, done